packages/atomic-angular/projects/atomic-angular/src/lib/atomic-angular.module.ts

```diff
@@ -0,0 +1,34 @@
+import {CommonModule} from '@angular/common';
+import {ModuleWithProviders, NgModule} from '@angular/core';
+
+import {
+  AtomicSearchInterface,
+  AtomicSearchBox,
+  AtomicResultList,
+  AtomicLoadMoreResults,
+} from './stencil-generated/components';
+
+import {defineCustomElements} from '@coveo/atomic/loader';
```

Comment:
This is to work around an issue with the stencil angular output, as described here https://github.com/ionic-team/stencil-ds-output-targets